src/widget/form/chatform.cpp

```diff
@@ -879,6 +889,52 @@ void ChatForm::loadHistory(QDateTime since, bool processUndelivered)
     chatWidget->verticalScrollBar()->setValue(savedSliderPos);
 }
 
+void ChatForm::onScreenshotCreate()
+{
+    QRect region;
+    int result = 0;
+
+    {
+        ScreenshotDialog screenshot(region);
+        result = screenshot.exec();
+    }
+
+    if (result == QDialog::Accepted)
+    {
+        QScreen *screen = QGuiApplication::primaryScreen();
+        if (screen)
+        {
+            // Have to wait for the rubber band to disappear.
+            // Using flush, sendPostedEvents and processEvents does not work.
+            QTimer::singleShot(100, this, [=]()
```

Comment:
This is quite the hack.... I'm not too comfortable with this, it's magical and literally just praying and hoping that it works.